pythonforandroid/build.py

```diff
@@ -764,7 +764,7 @@ def run_pymodules_install(ctx, modules, project_dir=None,
         shprint(host_python, '-m', 'venv', 'venv')
 
         # Prepare base environment and upgrade pip:
-        base_env = copy.copy(os.environ)
+        base_env = dict(copy.copy(os.environ))
```

Comment:
This has nothing to do with the macOS venv fix, but it suddenly started failing (develop branch included). So I included the fix in the PR but dedicated commit

Reply:
:+1: but you created a separate PR right #2160? nevermind, it's fine to me!